packages/common/src/services/explore/Explore.ts

```diff
@@ -97,18 +103,23 @@ export class Explore {
       const tracks = lineupItems.filter(
         (lineupItem): lineupItem is UserTrack => 'track_id' in lineupItem
       )
+      const { data, signature } = await this.audiusBackendInstance.signDiscoveryNodeRequest()
+      const history = await sdk.full.users.getUsersTrackHistory({
+        id: encodeHashId(currentUserId),
+        encodedDataMessage: data,
+        encodedDataSignature: signature,
+        limit: 100
+      })
+      const activityData = history.data as APIActivityV2[]
```

Comment:
JC, is the typing messed up here on sdk side?

Reply:
i realize i'm failing typecheck -- i think this is actually good now, will remove.